main/src/main/scala/sbt/internal/Inspect.scala

```diff
@@ -27,19 +27,18 @@ object Inspect {
   val Uses: Mode = UsesMode
   val Definitions: Mode = DefinitionsMode
 
-  def parser: State => Parser[(Inspect.Mode, ScopedKey[_])] =
+  def parser: State => Parser[() => String] =
     (s: State) =>
-      spacedModeParser(s) flatMap {
-        case opt @ (UsesMode | DefinitionsMode) =>
-          allKeyParser(s).map(key => (opt, Def.ScopedKey(Global, key)))
-        case opt @ (DependencyTreeMode | Details(_)) => spacedKeyParser(s).map(key => (opt, key))
+      spacedModeParser(s) flatMap { mode =>
+        commandHandler(s, mode) | keyHandler(s)(mode)
     }
-  val spacedModeParser: (State => Parser[Mode]) = (s: State) => {
+  val spacedModeParser: State => Parser[Mode] = (_: State) => {
+    val default = "-" ^^^ Details(false)
```

Comment:
@eed3si9n @dwijnand Is it a decent solution to wanting inspect the key named 'actual' with `Details(false)`? I can leave it out or renamed `-` to a different name if preferred.

Reply:
It's a bit ironic since `-` is an actual command name in sbt 0.13 to do early processing, like `-warn`, but I think `inspect - somekey` is a decent escaping solution.